resources/default_pipeline_environment.yml

```diff
@@ -84,6 +84,23 @@ steps:
         fail:
           high: '0'
     archive: false
+  cloudFoundryDeploy:
+    cloudFoundry:
+      apiEndpoint: 'https://api.cf.eu10.hana.ondemand.com'
```

Comment:
Strange to see that URL here. Is that URL always right? Would expect that this is specific to a customers landscape.

Reply:
I was under the assumption that this is the default url. If not, we should remove it.